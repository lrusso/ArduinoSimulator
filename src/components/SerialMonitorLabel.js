import React from "react"
import { t } from "../utils/service"

const SerialMonitorLabel = () => {
  return <div style={styles.label}>{t("SERIAL_MONITOR")}</div>
}

const styles = {
  label: {
    float: "left",
    fontFamily: "Arial",
    fontSize: "13px",
    lineHeight: 2,
    fontWeight: "bold",
    marginLeft: "7px",
    marginRight: "1px",
    cursor: "default",
  },
}

export default SerialMonitorLabel
