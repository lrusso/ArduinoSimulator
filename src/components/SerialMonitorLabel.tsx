import React from "react"
import { t } from "../utils/languages"

const SerialMonitorLabel = () => {
  return <div style={styles.label}>{t("SERIAL_MONITOR")}</div>
}

const styles: { [key: string]: React.CSSProperties } = {
  label: {
    float: "left",
    fontFamily: "'Segoe UI', Tahoma, Geneva, Verdana, sans-serif",
    fontSize: "13px",
    lineHeight: 2,
    fontWeight: "bold",
    marginLeft: "7px",
    marginRight: "1px",
    cursor: "default",
    color: "#AAA",
  },
}

export default SerialMonitorLabel
