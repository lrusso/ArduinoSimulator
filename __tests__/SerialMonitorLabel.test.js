import React from "react"
import renderer from "react-test-renderer"
import SerialMonitorLabel from "../src/components/SerialMonitorLabel"

describe("SerialMonitorLabel", () => {
  test("Rendering 1", () => {
    const tree = renderer.create(<SerialMonitorLabel />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
