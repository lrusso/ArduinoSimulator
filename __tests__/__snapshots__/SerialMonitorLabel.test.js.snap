// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SerialMonitorLabel Rendering 1 1`] = `
<div
  style={
    {
      "cursor": "default",
      "float": "left",
      "fontFamily": "Arial",
      "fontSize": "13px",
      "fontWeight": "bold",
      "lineHeight": 2,
      "marginLeft": "7px",
      "marginRight": "1px",
    }
  }
>
  SERIAL MONITOR
</div>
`;
